src/resty/ljsonschema/init.lua

```diff
@@ -195,7 +198,7 @@ end
 
 function codectx_mt:as_func(name, ...)
   local chunk = self:as_string()
-  local loaded_chunk, err = load(chunk, 'jsonschema:' .. (name or 'anonymous'))
+  local loaded_chunk, err = load(chunk, 'jsonschema:' .. (name or 'anonymous'), 'bt', custom_env)
```

Comment:
Does the `load()` support the extra argument `custom_env`?

Reply:
Ignore me.

```lua
load (ld [, source [, mode [, env]]])
```

Reply:
LuaJIT ported this feature from 5.2. See https://luajit.org/extensions.html